plotly/tools.py

```diff
@@ -2275,6 +2277,25 @@ def create_candlestick(open, high, low, close,
         return dict(data=data, layout=layout)
 
 
+    @staticmethod
+    def create_dendrogram(X, orientation="bottom", labels=None,
+                          colorscale=None, **kwargs):
+        """
+        Returns a dendrogram Plotly figure object.
+
+        X: Heatmap matrix as array of arrays
```

Comment:
Is a more common convention for matrices, "Z"? That's what we use in our heatmaps, but I'm not sure if there is a dendrogram convention that uses "X"